transforms/integration/transforms/set-value.js

```diff
@@ -67,6 +68,16 @@ function hasFluentTriggerCall(j, path) {
     ;
 }
 
+function hasFluentTriggerAndMoreCall(j, path) {
+  const greatGrandParent = path.parentPath.parentPath.parentPath.node;
```

Comment:
used `path.parentPath` as I think it allows safe access when traversing up the tree.